tools/test-generator/templates/bracket-push/template.ml

```diff
@@ -0,0 +1,16 @@
+open Core.Std
```

Comment:
I may have said this before, but probably haven't said this enough. Your work on the test generator is very valuable! Very good job.